runtime/src/ancient_append_vecs.rs

```diff
@@ -110,28 +109,6 @@ pub fn is_ancient(storage: &AppendVec) -> bool {
     storage.capacity() >= get_ancient_append_vec_capacity()
 }
 
-/// return true if the accounts in this slot should be moved to an ancient append vec
-/// otherwise, return false and the caller can skip this slot
-/// side effect could be updating 'current_ancient'
-pub fn should_move_to_ancient_append_vec(
```

Comment:
Was this fn not called from anywhere? Surprised to see a function getting removed but no callers updated.

Reply:
This whole file is dead code atm. The calling code is intertwined with accountsdb and is hardening now. This particular function needed to share more with the existing shrink path so it needed to move into accounts_db.rs itself. I was trying to move some code out of that file.